pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1786,14 +1821,14 @@ var _ = Describe("LaunchTemplate Provider", func() {
 				nodeClass.Spec.UserData = aws.String("special user data")
 				nodeClass.Spec.AMISelectorTerms = []v1beta1.AMISelectorTerm{{Tags: map[string]string{"*": "*"}}}
 				nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyCustom
-				awsEnv.EC2API.DescribeImagesOutput.Set(&ec2.DescribeImagesOutput{Images: []*ec2.Image{
+				nodeClass.Status.AMIs = []v1beta1.AMI{
 					{
-						Name:         aws.String(coretest.RandomName()),
-						ImageId:      aws.String("ami-123"),
-						Architecture: aws.String("x86_64"),
-						CreationDate: aws.String("2022-08-15T12:00:00Z"),
+						ID: "ami-123",
+						Requirements: scheduling.NewRequirements(
```

Comment:
Any reason we wouldn't be using NewNodeSelectorRequirements()?

Reply:
This mainly to use the `scheduling.NewRequirement`. This would staying consistent with how we define our default AMIs